### PR TITLE
Rules `unit-*` correctly handles browser hacks.

### PR DESCRIPTION
**Pull request by evilebottnawi**
Close https://github.com/stylelint/stylelint/issues/1645
/cc @davidtheclark @jeddy3

### CODE REVIEW

File: src/utils/getUnitFromValueNode.js [hunk 1]
```diff
@@ -15,6 +15,9 @@ export default function (node) {
   if (!node || (node && !node.value)) { return null }
 
   const value = blurInterpolation(node.value, "")
+    // ignore hack unit
+    .replace("\\0", "")
+    .replace("\\9", "")
```
Comment (davidtheclark) [inline src/utils/getUnitFromValueNode.js]: Should this be a part of every unit identifying rule, or just this one?

Comment (evilebottnawi) [inline src/utils/getUnitFromValueNode.js]: @davidtheclark poorly understood, we remove all possible hack where parsed unit

Comment (davidtheclark) [inline src/utils/getUnitFromValueNode.js]: Whoops, somehow I misread the filename. Looks like this is done in the right place!

**Comment by davidtheclark** [comment]
LGTM!

**Comment by jeddy3** [comment]
@evilebottnawi Nice.

As an aside, it's kind of odd to see, after a seemingly endless sequence of fixes for non-standard syntax, a fix for simple *CSS hack*. It's a refreshing change :)

**Pull request merged**